src/operations/channels.jl

```diff
@@ -0,0 +1,323 @@
+"""
+    SplitChannels <: Augmentor.Operation
+
+Description
+--------------
+
+Splits out the color channels of the given image using the
+function `ImageCore.channelview`. This will effectively create a
+new array dimension for the colors in the front. In contrast to
+`ImageCore.channelview` it will also result in a new dimension
+for Gray images.
```

Comment:
Here is for example for how Knet thinks of "channels" and that it assumes that such a concept is present https://github.com/denizyuret/Knet.jl/blob/master/src/conv.jl#L98-L112